### PR TITLE
docs: Landing page and Logo Update

### PR DESCRIPTION
**Pull request by dat-adi**
## Changelog
- [x] Improved landing page.
- [x] Addition of the new ForgeFlux Logo into the Documentation.

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -3,43 +3,53 @@ title = "Magic Bridges"
 insert_anchor_links = "right"
 +++
 
-# ForgeFlux
+## A day of the past
 
-[ForgeFlux](https://forgeflux.org/) aims to implement federation in [software forges](https://en.wikipedia.org/wiki/Forge_(software)) , enabling
-a decentralized development environment.
+It's a Friday evening, and you've sat down to work on some code. \
```
Comment (realaravinth) [inline content/_index.md]: Can you remove `\` from all these documents?

**Review by realaravinth — CHANGES_REQUESTED** [review]
`\` seems to be there at the end of all sentences and it seems to [creep into the rendered content](https://github.com/dat-adi/docs/blob/main/content/_index.md)

File: content/_index.md (hunk 1 above)
Comment (dat-adi) [reply content/_index.md]: That might result in extremely large paragraphs, we cool with that?

The backslashes function as reasonable line breakers.

File: config.toml [hunk 2]
```diff
@@ -8,6 +8,6 @@ highlight_code = true
 highlight_theme = "base16-ocean-light"
 
 [extra]
-logo = "https://easydocs.codeandmedia.com/logo.svg"
+logo = "https://github.com/forgeflux-org/docs/blob/main/static/logo.svg"
```
Comment (realaravinth) [inline config.toml]: please change this URL to `/logo.svg`, all files in `/static` dir are available at root

File: static/logo.svg [hunk 3]
```diff
@@ -1 +1 @@
...
...-a6b1-41a4-97ac-f5e00be0ef3e" x="70.09" y="68.78" width="38.79" height="10.38" rx="1.94"/><rect class="be9c742b-a6b1-41a4-97ac-f5e00be0ef3e" x="70.09" y="87.47" width="24.59" height="10.38" rx="1.94"/><rect class="ab3dba79-30e8-459a-a0be-ce4217101439" x="104.96" y="87.47" width="24.59" height="10.38" rx="1.94"/><rect class="be9c742b-a6b1-41a4-97ac-f5e00be0ef3e" x="70.09" y="106.01" width="10.38" height="10.38" rx="1.94"/><rect class="ab3dba79-30e8-459a-a0be-ce4217101439" x="119.17" y="68.78" width="10.38" height="10.38" rx="1.94"/><rect class="ab3dba79-30e8-459a-a0be-ce4217101439" x="90.76" y="106.01" width="38.79" height="10.38" rx="1.94"/><text class="a7dc26de-4b77-4d05-9982-f373222849ac" transform="translate(142.91 113.34)"><tspan class="fe344c9d-5a72-4e1f-9d1e-725524f3af52">F</tspan><tspan class="e2050f92-abb6-4b6b-a25b-419347361e9c" x="34.98" y="0">ORGE</tspan><tspan class="abea20a3-df56-4a97-97f6-6eaf4c409d82" x="165" y="0">F</tspan><tspan class="f47252b2-8628-4792-bd7e-e4346d71e80c" x="199.98" y="0">L</tspan><tspan class="e32e9238-6004-409e-a320-45826ebea3e8" x="225.24" y="0">U</tspan><tspan class="b5a0f27e-7f61-4ece-80f2-d512df9640a8" x="259.24" y="0">X</tspan></text></svg>
```
Comment (realaravinth) [inline static/logo.svg]: The logo styling changes in different zoom levels, can you contact Sai Pranay and get this fixed?

https://user-images.githubusercontent.com/33521410/150632830-311dd2db-64d0-4a08-9420-ca9fc459594b.mp4

**Review by realaravinth — CHANGES_REQUESTED** [review]

File: config.toml (hunk 2 above)
Comment (dat-adi) [reply config.toml]: Ah, very well. That will be done.

File: static/logo.svg (hunk 3 above)
Comment (dat-adi) [reply static/logo.svg]: Interesting. Sure, I'll get in touch with him.

Comment (dat-adi) [reply static/logo.svg]: The issue has been resolved.

**Comment by dat-adi** [comment]
The sidebar is now displayed with an appropriately sized logo as compared to the initial draft.
image

**Review by realaravinth — APPROVED** [review]

**Pull request merged**